projects/github-actions/repo-gardening/src/tasks/triage-new-issues/index.js

```diff
@@ -19,21 +19,41 @@ async function hasPriorityLabels( octokit, owner, repo, number ) {
 }
 
 /**
- * Find specific plugin impacted by issue, based off issue contents.
+ * Find list of plugins impacted by issue, based off issue contents.
  *
  * @param {string} body - The issue content.
- * @returns {string} Plugin concerned by issue.
+ * @returns {Array} Plugins concerned by issue.
  */
-function findPlugin( body ) {
-	const regex = /###\sImpacted\splugin\n\n(\w*)\n/gm;
+function findPlugins( body ) {
+	const regex = /###\sImpacted\splugin\n\n([a-zA-Z ,]*)\n\n/gm;
 
 	let match;
 	while ( ( match = regex.exec( body ) ) ) {
-		const [ , plugin ] = match;
-		return plugin;
+		const [ , plugins ] = match;
+		return plugins.split( ', ' );
 	}
 
-	return null;
+	debug( `triage-new-issues: No plugin indicators found.` );
+	return [];
+}
+
+/**
+ * Find platform info, based off issue contents.
+ *
+ * @param {string} body - The issue content.
+ * @returns {Array} Platforms impacted by issue.
+ */
+function findPlatforms( body ) {
+	const regex = /###\sPlatform\s\(Simple,\sAtomic,\sor\sboth\?\)\n\n([a-zA-Z ,-]*)\n\n/gm;
+
+	let match;
+	while ( ( match = regex.exec( body ) ) ) {
```

Comment:
Seems odd to be using a `while` when you always exit on the first time through.
```suggestion
	const match = regex.exec( body );
	if ( match ) {
```
Same where you copy-pasted this from above, for that matter.

Reply:
👍 Done in eb9c1ce99ca99a52eb60a02d5b91097cf782b8f2